docs/Admin.md

```diff
@@ -349,24 +345,9 @@ const App = () => (
 
 You can also disable it completely along with the `/login` route by passing `false` to this prop.
 
-See The [Authentication documentation](./Authentication.md#customizing-the-login-and-logout-components) for more details.
```

Comment:
why did you remove this link?.

Reply:
I didn't, I just separated the linked paragraph in two distinct ones